shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/TripDetailsStopList.kt

```diff
@@ -13,7 +13,7 @@ constructor(val tripId: String, val stops: List<Entry>, val startTerminalEntry:
     data class Entry(
         val stop: Stop,
         val stopSequence: Int,
-        val alert: Alert?,
+        val disruption: RealtimePatterns.Format.Disruption?,
```

Comment:
This data class is too big already, so I didn't want to add another field to it; since the `Disruption` now contains both the `alert` and the `iconName`, and since we're calling the same `UpcomingTripView` widget that we'd call from nearby transit, I figured I'd just reuse `RealtimePatterns.Format.Disruption` directly.